plasmapy/particles/particle_class.py

```diff
@@ -368,7 +333,13 @@ def __init__(
             if self.element and self._attributes['integer charge']:
                 categories.add('ion')
 
-            # Element properties
+##########################
+
```

Comment:
Delete lines...[PEP8: there should be a single blank line between methods](https://www.python.org/dev/peps/pep-0008/#blank-lines)